### ISSUE TITLE
Ubuntu Client error: No route to host

### ISSUE THREAD

**Issue by tienloc1** [debauchee/barrier#1121]
It's working fine for me yesterday but it's stopped now.

Server: Windows 10 - Server IP: 192.168.1.6
Firewall - Inbound: Added port 24800 - Done
`
[2021-04-08T09:47:09] INFO: starting server
[2021-04-08T09:47:09] INFO: config file: C:/Users/Loc/AppData/Local/Temp/Barrier.APCXXT
[2021-04-08T09:47:09] INFO: log level: INFO
[2021-04-08T09:47:09] INFO: service command updated
[2021-04-08T09:47:09] INFO: starting new process as privileged user
[2021-04-08T09:47:09] ERROR: failed to get desktop path, no drop target available, error=2
started server (IPv4/IPv6), waiting for clients
server status: active
`
Test-NetConnection result:
`
ComputerName   : 192.168.1.6
RemoteAddress    : 192.168.1.6
RemotePort       : 24800
InterfaceAlias   : Ethernet
SourceAddress    : 192.168.1.6
TcpTestSucceeded : True
`

Client: Ubuntu 20.04 - Client IP: 192.1.68.1.5
`
[2021-04-08T09:47:49] INFO: starting client
[2021-04-08T09:47:49] INFO: config file: /tmp/Barrier.Gkkruu
[2021-04-08T09:47:49] INFO: log level: INFO
[2021-04-08T09:47:49] NOTE: started client
[2021-04-08T09:47:49] NOTE: connecting to '192.168.1.6': 192.168.1.6:24800
[2021-04-08T09:47:49] INFO: OpenSSL 1.1.1f  31 Mar 2020
[2021-04-08T09:47:52] WARNING: failed to connect to server: No route to host
`

I'm not sure what happened and how I can resolve it now :(.

**Comment by tienloc1** [comment]
It's working again now, but the server IP changed 192.168.1.6 to 192.168.1.3, I'm not sure why the problem resolved by itself but seems it's related to the server IP.

**Comment by tienloc1** [comment]
The problem repeated again many times :(

**Comment by stonespheres** [comment]
> The problem repeated again many times :(

Are you using DHCP? 



> It's working again now, but the server IP changed 192.168.1.6 to 192.168.1.3, I'm not sure why the problem resolved by itself but seems it's related to the server IP.
I suspect you're getting a new lease from your router to your server, which requires you to change the server IP address in Barrier. You can check your server IP with:

```
ip r
```
You may also be able to specify the host name in Barrier, but I'm not 100% sure about this. To find the hostname of your server:
```
hostname
```

**Comment by ERasnier** [comment]
Yeah the client connects with the server ip entered and nothing else. You'd have to re-enter the new server ip every time the server resets.

**Comment by stonespheres** [comment]
> Yeah the client connects with the server ip entered and nothing else. You'd have to re-enter the new server ip every time the server resets.

It should be possible to start the client instance *from the server.* I do it this way when I want to access my RPi.

**Comment by ERasnier** [comment]
I'm getting a refused connection from the ssh command but on the server logs it's saying that client connection has been accepted.
